<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-B8PP1IT" failures="0" tests="9" name="testNgPackage.DependencyClassTwo" time="0.007" errors="2" timestamp="2022-09-07T22:53:13 EDT" skipped="7">
  <testcase classname="testNgPackage.DependencyClassTwo" name="afterTest" time="0.001">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testNgPackage.DependencyClassTwo.afterTest(DependencyClassTwo.java:146)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.afterRun(TestRunner.java:877)
at org.testng.TestRunner.run(TestRunner.java:599)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- afterTest -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="ClickingOnShopButton" time="0.000">
    <skipped/>
  </testcase> <!-- ClickingOnShopButton -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="openingBrowser" time="0.005">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testNgPackage.DependencyClassTwo.openingBrowser(DependencyClassTwo.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- openingBrowser -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="addingBooksInBasket" time="0.000">
    <skipped/>
  </testcase> <!-- addingBooksInBasket -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="ClickOnHomeButton" time="0.000">
    <skipped/>
  </testcase> <!-- ClickOnHomeButton -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="MoveToThreeSliders" time="0.000">
    <skipped/>
  </testcase> <!-- MoveToThreeSliders -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="clickingImageInArrivals" time="0.000">
    <skipped/>
  </testcase> <!-- clickingImageInArrivals -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="OpningUrl" time="0.000">
    <skipped/>
  </testcase> <!-- OpningUrl -->
  <system-out/>
  <testcase classname="testNgPackage.DependencyClassTwo" name="clickingOnReviewsTab" time="0.001">
    <skipped/>
  </testcase> <!-- clickingOnReviewsTab -->
  <system-out/>
</testsuite> <!-- testNgPackage.DependencyClassTwo -->
